lib/commands/destroy.js

```diff
@@ -22,6 +23,20 @@ module.exports = Command.extend({
   anonymousOptions: [
     '<blueprint>'
   ],
+  
+  beforeRun: function(rawArgs){
+    // merge in blueprint availableOptions
+    var blueprint;
+    var debug = require('debug')('ember-cli/commands/destroy');
+    try{
+      blueprint = this.lookupBlueprint(rawArgs[0]);
+      this.registerOptions( blueprint );
+    }
+    catch(e) {
+      // ignore this error, invalid blueprints are handled in run
```

Comment:
Sure, I believe it made you nervous when I did the same thing for the generate command. I'll apply the same changes to it as well.